Resources/config/services.xml

```diff
@@ -12,7 +12,7 @@
             <argument>%coop_tilleuls_ovh.consumer_key%</argument>
         </service>
 
-        <service id="ovh" alias="coop_tilleuls_ovh.api" />
+        <service id="ovh" alias="coop_tilleuls_ovh.api" public="true" />
```

Comment:
It would be great to also add an alias to `\Ovh\Api` to enable auto-wiring.

Reply:
Done. Sorry for the delay.